Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
```

Comment:
> (A patch to the makefile would be a bit excessive, and violate KISS, in my opinion.)

Patches are good when they are submitted upstream as they may be included in future versions. Can you please submit a patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file so we can use `make install` and the library is named correctly. Thanks!